permissions/plugin-tanaguru.yml

```diff
@@ -0,0 +1,11 @@
+---
+name: "tanaguru"
+github: "jenkinsci/tanaguru-plugin"
+paths:
+- "org/jenkins-tanaguru/plugins/tanaguru"
```

Comment:
This does not seem to be an actual domain name.

Reply:
"Run" logs told us user "tanaguru" doen't exist in Artifactory, but i'm logged with this same user. I don't really understand what is happening here.

Reply:
Probably just a cache.